### PR TITLE
all tests passed

### PR DESCRIPTION
**Pull request by KrisBot2000**

### CODE REVIEW

File: secret_number.js [hunk 1]
```diff
@@ -1,4 +1,40 @@
+//var secretNumber = function(){
 'use strict';
 module.exports = function() {
-  
+
+
+var sameRandomNumber = 0;
+var genRandomNumber = Math.random();
```
Comment (taesup) [inline secret_number.js]: This could be simplified. I don't think you need both sameRandomNumber and genRandomNumber.

File: secret_number.js [hunk 2]
```diff
@@ -1,4 +1,40 @@
...
+
+  return function(){
+  sameRandomNumber = genRandomNumber*1000001;
```
Comment (taesup) [inline secret_number.js]: So at this point, genRandomNumber is Math.random() * 1000001 which at the highest is 1000000.99999 which is slightly above the limit of 1000000. Can you think of any other Math method that will allow you to hit 1000000 exactly?

**Review by taesup — CHANGES_REQUESTED** [review]

File: secret_number.js (hunk 1 above)
Comment (KrisBot2000) [reply secret_number.js]: fixed

Comment (taesup) [reply secret_number.js]: did you push your fix?

**Comment by taesup** [comment]
Looks good! Free feel to merge into master.

Thumbs Up

**Pull request merged**